packages/statemanager/src/stateManager.ts

```diff
@@ -155,6 +155,7 @@ export class DefaultStateManager implements EVMStateManagerInterface {
   protected _debug: Debugger
   protected _accountCache?: AccountCache
   protected _storageCache?: StorageCache
+  protected _codeCache2?: CodeCache
```

Comment:
Rename to `_codeCache`?

Reply:
I removed the old cache and renamed the new one in its place.